tests/cli/workflow/rnafusion/test_cli_rnafusion_run.py

```diff
@@ -72,10 +72,8 @@ def test_without_config(
     case_id: str = rnafusion_case_id
     # WHEN dry running with dry specified
     result = cli_runner.invoke(run, [case_id, "--dry-run"], obj=rnafusion_context)
-    # THEN command should NOT execute successfully
-    assert result.exit_code != EXIT_SUCCESS
-    # THEN warning should be printed that no config file is found
-    assert "No config file found" in caplog.text
+    # THEN command should execute successfully (dry-run)
+    assert result.exit_code == EXIT_SUCCESS
```

Comment:
Should we add a test with no dry-run to make sure it fails if no config is found?